parm/config/gefs/config.base

```diff
@@ -75,6 +75,7 @@ export NMV="/bin/mv"
 export NLN="/bin/ln -sf"
 export VERBOSE="YES"
 export KEEPDATA="NO"
+export DEBUG="NO" # Only works on WCOSS2
```

Comment:
`DEBUG_SCHEDULER` is perhaps better suited than a generic `DEBUG`. What do you think?

Reply:
Have changed to `DEBUG_SCHEDULER` per reviewer suggestion and offline discussion.